.travis.yml

```diff
@@ -16,6 +16,11 @@ env:
   - secure: Qi+QPGx8olG4Xu5xa4LZPP1yMTiRtyEbXgW8Tr2XBPRQO7DL8wjmC0wgOig0+UdmtMBAoS9QC9qnLl06/Ry6Bppsdg1kdNp71kQLU7BCDz6aJekpatpoUJ2C4KGhNXVcBEX3v7rH1zlUVwDfcrjD2tgH4lRINOd7O8THuo6fA9XiRAXJHImukXtbcoRHjqBZSq7wAHL9IsdR9GaRTx40P6sXoJeRaP/PBw+diZkAO8WSDxaVgdINVyNXRiCfXiGppp4ql1C4tuFfpnERsbEUGrQ+vttOS5x8Xj/5MDpuYts1hKUfxX+dA7C/LYj131/2c0eKvCfNGKhzO/aKV5Ucf5k2+0wWOMAPrH788d3nvthqNY2UJOlq/Qqh7NXQBVy0BEcDCGGzjYXv39vxNJ2YUaJTmLGpwo3FXHDNSm7FXol9oN31Sp0vkT78zgvBemplbB4KsJ1jmWzvJGatqjbP3POCrU6jM7wLg3EsRDLeKyJf+oxgd63ZMD3wCKXAjaKs/0EkTz4R0a8aIBzdC/rDIAQHORy/92Cuta7UGPCHIqUSvrmWr7yf8y97rygmG2OB41yx5eVL7mPnt2e6oxEyRdENM3D7lYpGfynwprT+p9f3pBxtpaztL/E52JvGg1hlTzC/Ng+EZDO85iUegzzFYzCSJkivoBHJIuyUpvib4vA=
 jobs:
   include:
+  - stage: test
+    script:
+    - node ./index.js &
+    - sleep 2m
+    - node ./smoke_test.js
```

Comment:
Confusion between Tests (Unit/Integration) with, what we are calling smoke tests, that verifies if built container has started.